cluster/pulumi/canton-network/src/info.ts

```diff
@@ -73,7 +71,7 @@ export function installInfo(
           ? {
               chainIdSuffix: getChainIdSuffix(),
               migrationId: decentralizedSynchronizerMigrationConfig.upgrade.id,
-              version: cnChartVerstionToString(
+              version: CnChartVersion.stringify(
```

Comment:
```suggestion
              version: CnChartVersion.toString(
```

some bikeshedding; we definitely use the `toString` wording in other places, whereas the `stringify` one sounded alien to me (like from some external library)

Reply:
I'd agree if we were talking about a method. `version.toString()` would definitely fit the JS/TS ecosystem. Since we're using a simple structural type here we cannot add methods. In JS there is for example `JSON.stringify` so that's where I took inspiration. Since you're more familiar with the codebase I'm willing to trust you that `toString` might be a better fit but I wanted to present my reasoning.